packages/kit/src/utils/escape.js

```diff
@@ -25,42 +25,40 @@ const escape_json_string_in_html_dict = {
  * @param {string} str
  */
 export function escape_json_in_html(str) {
-	return escape(str, escape_json_in_html_dict, (code) => `\\u${code.toString(16).toUpperCase()}`);
+				// adapted from https://github.com/vercel/next.js/blob/694407450638b037673c6d714bfe4126aeded740/packages/next/server/htmlescape.ts
+				// based on https://github.com/zertosh/htmlescape
+				// License: https://github.com/zertosh/htmlescape/blob/0527ca7156a524d256101bb310a9f970f63078ad/LICENSE
+				/**
+				 * @type { Record<string, string> }
+				 */
+				 const escape_lookup = {
+					'&': '\\u0026',
+					'>': '\\u003e',
+					'<': '\\u003c',
+					'\u2028': '\\u2028',
+					'\u2029': '\\u2029'
+				};
```

Comment:
can we hoist this like the other dictionaries, so we're not recreating this object unnecessarily?

Reply:
done